2016-04-11 12:12:20.123 [info] <0.7.0> Application lager started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:20.127 [info] <0.7.0> Application global_table started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:20.376 [info] <0.7.0> Application mnesia started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:21.853 [info] <0.7.0> Application pangzi started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:21.857 [info] <0.7.0> Application esqlite started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:38.339 [info] <0.7.0> Application config started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:38.344 [info] <0.7.0> Application cron started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:38.345 [info] <0.7.0> Application lc started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:38.347 [info] <0.7.0> Application system_broadcast started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:38.392 [error] <0.738.0> gen_server auction_svr terminated with reason: no function clause matching lists:keyreplace3(2, 1, none, {2,true}) line 704
2016-04-11 12:12:38.393 [error] <0.738.0> CRASH REPORT Process auction_svr with 0 neighbours exited with reason: no function clause matching lists:keyreplace3(2, 1, none, {2,true}) line 704 in gen_server:terminate/6 line 746
2016-04-11 12:12:38.394 [error] <0.732.0> Supervisor com_mod_sup had child auction_svr started with auction_svr:start_link() at <0.738.0> exit with reason no function clause matching lists:keyreplace3(2, 1, none, {2,true}) line 704 in context child_terminated
2016-04-11 12:12:38.394 [error] <0.756.0> gen_server auction_svr terminated with reason: no function clause matching lists:keyreplace3(2, 1, none, {2,true}) line 704
2016-04-11 12:12:38.395 [error] <0.756.0> CRASH REPORT Process auction_svr with 0 neighbours exited with reason: no function clause matching lists:keyreplace3(2, 1, none, {2,true}) line 704 in gen_server:terminate/6 line 746
2016-04-11 12:12:38.395 [error] <0.732.0> Supervisor com_mod_sup had child auction_svr started with auction_svr:start_link() at <0.756.0> exit with reason no function clause matching lists:keyreplace3(2, 1, none, {2,true}) line 704 in context child_terminated
2016-04-11 12:12:38.395 [error] <0.732.0> Supervisor com_mod_sup had child auction_svr started with auction_svr:start_link() at <0.756.0> exit with reason reached_max_restart_intensity in context shutdown
2016-04-11 12:12:38.414 [error] <0.731.0> Supervisor mobile_game_sup had child com_mod_sup started with com_mod_sup:start_link() at <0.732.0> exit with reason shutdown in context child_terminated
2016-04-11 12:12:38.445 [error] <0.788.0> gen_server auction_svr terminated with reason: no function clause matching lists:keyreplace3(2, 1, none, {2,true}) line 704
2016-04-11 12:12:38.445 [error] <0.788.0> CRASH REPORT Process auction_svr with 0 neighbours exited with reason: no function clause matching lists:keyreplace3(2, 1, none, {2,true}) line 704 in gen_server:terminate/6 line 746
2016-04-11 12:12:38.453 [info] <0.7.0> Application crypto started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:38.454 [error] <0.781.0> Supervisor com_mod_sup had child auction_svr started with auction_svr:start_link() at <0.788.0> exit with reason no function clause matching lists:keyreplace3(2, 1, none, {2,true}) line 704 in context child_terminated
2016-04-11 12:12:43.454 [warning] <0.6.0> lager_error_logger_h dropped 59 messages in the last second that exceeded the limit of 50 messages/sec
2016-04-11 12:12:43.454 [error] <0.806.0> CRASH REPORT Process <0.806.0> with 0 neighbours exited with reason: {error,"timed out"} in gen_server:init_it/6 line 322
2016-04-11 12:12:43.456 [info] <0.7.0> Application mobile_game started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:43.464 [info] <0.7.0> Application mobile_game exited with reason: shutdown
2016-04-11 12:12:43.465 [info] <0.7.0> Application gm_controller_server started on node 'fengzhu@192.168.0.254'
2016-04-11 12:12:51.516 [error] <0.861.0> Supervisor game had child {acceptor,game,1} started with lc_acceptor:start_link(#Port<0.6105>, #Fun<player_sup.start_child.1>) at <0.862.0> exit with reason no such process or port in call to gen_server:call(player_sup, {start_child,[#Port<0.6111>]}, infinity) in context child_terminated
2016-04-11 12:17:35.332 [error] <0.861.0> Supervisor game had child {acceptor,game,2} started with lc_acceptor:start_link(#Port<0.6105>, #Fun<player_sup.start_child.1>) at <0.863.0> exit with reason no such process or port in call to gen_server:call(player_sup, {start_child,[#Port<0.6116>]}, infinity) in context child_terminated
2016-04-11 12:17:56.298 [error] <0.861.0> Supervisor game had child {acceptor,game,3} started with lc_acceptor:start_link(#Port<0.6105>, #Fun<player_sup.start_child.1>) at <0.864.0> exit with reason no such process or port in call to gen_server:call(player_sup, {start_child,[#Port<0.6117>]}, infinity) in context child_terminated
